generators/app/templates/gitignore

```diff
@@ -3,7 +3,6 @@ coverage
 dist
 .opt-in
 .opt-out
-.DS_Store
```

Comment:
Let's keep this one. But remove the `.opt-*` stuff above. That's legacy stuff I don't use anymore.

Reply:
The `.opt-*` stuff is still mentioned in the `CONTRIBUTING` file and is still used by `kcd-scripts` tho.

https://github.com/kentcdodds/kcd-scripts/blob/72e81af07c4c22345ae362aed55ff98402601cf3/src/scripts/pre-commit.js#L26

Reply:
Oh yeah. Let's remove that feature instead. I don't think it's really useful or something I want supported.

Reply:
I'll create a  PR for `kcd-scripts` that removes the feature 🙂

Reply:
https://github.com/kentcdodds/kcd-scripts/pull/119